engine/dlib/src/dmsdk/dlib/image.h

```diff
@@ -113,6 +126,28 @@ namespace dmImage
      * @return data [type: uint32_t] the image height
      */
     const void* GetData(HImage image);
+
+    /*#
+     * Get the block size
+     * @param mem [type: void*] the .astc memory (including the header)
+     * @param memsize [type: uint32_t] the length of the memory blob
+     * @param width [type: uint32_t*] the block width
+     * @param height [type: uint32_t*] the block height
+     * @param depth [type: uint32_t*] the block depth
+     * @return result [type: bool] true if it's an astc file
+     */
+    bool GetAstcBlockSize(const void* mem, uint32_t memsize, uint32_t* width, uint32_t* height, uint32_t* depth);
```

Comment:
The astc format is non-trivial to unpack, so we stick with helper functions for the header.